.gitignore

```diff
@@ -22,6 +22,8 @@ contrib/babelfishpg_tsql/antlr/cmake_install.cmake
 contrib/babelfishpg_tsql/pl_reserved_kwlist_d.h
 contrib/babelfishpg_tsql/pl_unreserved_kwlist_d.h
 contrib/babelfishpg_tsql/sql/babelfishpg_tsql--[0-9].[0-9].[0-9]--[0-9].[0-9].[0-9].sql
+contrib/babelfishpg_tsql/sql/babelfishpg_tsql--[0-9].[0-9].[0-9]--[0-9].[0-9][0-9].[0-9].sql
+contrib/babelfishpg_tsql/sql/babelfishpg_tsql--[0-9].[0-9][0-9].[0-9]--[0-9].[0-9][0-9].[0-9].sql
```

Comment:
Thanks for adding these pattern in ignore files.
Non-blocking: Can't we have something genric like digit*.digit*.digit*, although this is also fine.

Reply:
Thanks for the suggestion, updated the expression.